controllers/bootstrap/init.go

```diff
@@ -457,3 +462,39 @@ func (b *Bootstrap) deleteSubscription(name, namespace string) error {
 
 	return nil
 }
+
+func (b *Bootstrap) waitOperatorReady(name, namespace string) error {
+	if err := utilwait.PollImmediate(time.Second*10, time.Minute*10, func() (done bool, err error) {
```

Comment:
Do we need to use named return value for this anonymous function?

Reply:
I want me to be clear about the meaning of this Boolean value.

Reply:
Otherwise, I believe readers may be confused about if the boolean means `continue` or `finish`.

Reply:
Get it!